src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -135,10 +135,10 @@
   <data name="AvoidAssemblyGetFilesInSingleFileMessage" xml:space="preserve">
     <value>'{0}' will throw for assemblies embedded in a single-file app</value>
   </data>
-  <data name="RequiresAssemblyFilesMessage" xml:space="preserve">
-    <value />
-  </data>
   <data name="RequiresAssemblyFilesTitle" xml:space="preserve">
-    <value />
+    <value>Avoid calling members marked with `RequiresAssemblyFilesAttribute` when publishing as a single-file</value>
```

Comment:
Nit: can we change (here and above) the backticks for common apostrophe (') similar to

https://github.com/mono/linker/blob/3c3771c5f0818f9b88387bd2bc1d9e58712e6b31/src/ILLink.RoslynAnalyzer/Resources.resx#L124

Or https://github.com/mono/linker/blob/master/src/linker/Resources/Strings.resx ?

Reply:
Is calling the right terminology here?. Are we going to add a new message for each bundle like format?

Reply:
> Is calling the right terminology here?

I'm changing the message to match something similar to https://github.com/mono/linker/pull/1778#discussion_r564429652
> Are we going to add a new message for each bundle like format?

Since they all will use the same custom attribute I would expect to print the same message independent of the kind of bundle